test/mocha/40-credential-status.js

```diff
@@ -0,0 +1,385 @@
+    id: 'urn:uuid:4886029a-7925-11ea-9274-10bf48838a41',
+    'example:test': 'foo'
+  },
+  issuanceDate: '2022-01-11T19:23:24Z',
+  credentialStatus: {
+    id: slCredential.id,
+    type: 'RevocationList2021Status',
+    statusListIndex: '67342',
+    statusListCredential: slCredential.id
+  },
+  issuer: slCredential.issuer,
+};
+
+// Revoked Status List 2021 Credential
+let revokedSlCredential = clone(slCredential);
+
+revokedSlCredential.credentialSubject.encodedList =
+  encodedList100KWith50KthRevoked;
+
+const revokedUnsignedCredential = clone(unsignedCredentialSl2021Type);
+revokedUnsignedCredential.credentialStatus.id =
+  `${revokedSlCredential.id}/50000`;
+revokedUnsignedCredential.credentialStatus.statusListIndex = 50000;
+revokedUnsignedCredential.credentialStatus.statusListCredential =
+  `${revokedSlCredential.id}/50000`;
+revokedUnsignedCredential.issuer = revokedSlCredential.issuer;
+
+// Revocation List 2020 Credential
+let rlCredential = {
+  '@context': [
+    'https://www.w3.org/2018/credentials/v1',
+    VC_RL_CONTEXT_URL
+  ],
+  id: `https://localhost:${PORT}/status/2`,
+  issuer: 'did:key:z6Mktpn6cXks1PBKLMgZH2VaahvCtBMF6K8eCa7HzrnuYLZv',
+  issuanceDate: '2022-01-10T04:24:12.164Z',
+  type: ['VerifiableCredential', 'RevocationList2020Credential'],
+  credentialSubject: {
+    id: `https://localhost:${PORT}/2/list`,
+    type: 'RevocationList2020',
+    encodedList: encodedList100k
+  }
+};
+
+// Unsigned 2020 Credential
+const unsignedCredentialRL2020Type = {
+  '@context': [
+    'https://www.w3.org/2018/credentials/v1',
+    VC_RL_CONTEXT_URL,
+    'https://w3id.org/security/suites/ed25519-2020/v1'
+  ],
+  id: 'urn:uuid:a0418a78-7924-11ea-8a23-10bf48838a41',
+  type: ['VerifiableCredential', 'example:TestCredential'],
+  credentialSubject: {
+    id: 'urn:uuid:4886029a-7925-11ea-9274-10bf48838a41',
+    'example:test': 'foo'
+  },
+  issuanceDate: '2022-01-11T19:23:24Z',
+  credentialStatus: {
+    id: rlCredential.id,
+    type: 'RevocationList2020Status',
+    revocationListIndex: '67342',
+    revocationListCredential: rlCredential.id
+  },
+  issuer: rlCredential.issuer,
+};
+
+// Revoked Revocation List 2020 Credential
+let revokedRlCredential = clone(rlCredential);
+
+revokedRlCredential.credentialSubject.encodedList =
+  encodedList100KWith50KthRevoked;
+
+const revokedUnsignedCredential2 = clone(unsignedCredentialRL2020Type);
+revokedUnsignedCredential2.credentialStatus.id =
+  `${revokedRlCredential.id}/50000`;
+revokedUnsignedCredential2.credentialStatus.revocationListIndex = 50000;
+revokedUnsignedCredential2.credentialStatus.revocationListCredential =
+  `${revokedRlCredential.id}/50000`;
+revokedUnsignedCredential2.issuer = revokedRlCredential.issuer;
+
+// mount the test routes
+app.get('/status/1',
+  // eslint-disable-next-line no-unused-vars
+  (req, res, next) => {
+    // responds with a valid status list 2021 type credential
+    res.json(slCredential);
+  });
+app.get('/status/1/50000',
```

Comment:
This approach seems to defeat the privacy purpose of the list design and may be flagged as such -- or erroneously copied by someone else. We may even want to reject status IDs that do this sort of thing and require them to use hash fragments instead.